src/pages/Detailed/header/DetailedHeader.module.scss

```diff
@@ -26,22 +44,50 @@
 
 .logo {
   display: flex;
-  flex-direction: column;
-  margin-right: 20px;
+  flex-direction: row;
+}
+
+.signalButton {
+  border: 1px solid var(--jungle-green) !important;
+}
+
+.watchlistButton {
+  margin-left: 16px;
+}
+
+.icon {
+  margin-right: 5px;
+}
+
+.watchlistIcon {
+  fill: var(--white);
+}
+
+.right {
+  margin-right: 15px;
+  margin-top: 15px;
 }
 
 .name {
   display: flex;
   flex-direction: column;
   justify-content: left;
-  padding-right: 150px;
+  margin-left: 16px;
+  width: 350px;
+}
+
+.projectSelector {
+  display: flex;
+  flex-direction: row;
 }
 
 .name h1 {
   margin: 0 !important;
   margin-right: 5px;
-  font-size: 24px !important;
+  font-size: 22px !important;
   line-height: 1em !important;
```

Comment:
`font-size` and `line-height` are assigned in the `text` mixin

Reply:
Yes. But here used '!important' because h1 override text mixin

Reply:
Styles should not be applied using `descendant selector` in case if it's possible to use `class selector` directly